REST-API-ADVANCED/advanced/src/main/java/com/epam/esm/springapp/AdvancedApplication.java

```diff
@@ -0,0 +1,135 @@
+package com.epam.esm.springapp;
+
+import com.epam.esm.certificate.entity.Certificate;
+import com.epam.esm.certificate.repo.CertificateRepository;
+import com.epam.esm.order.entity.Order;
+import com.epam.esm.order.repository.OrderRepository;
+import com.epam.esm.tag.entity.Tag;
+import com.epam.esm.tag.repo.TagRepository;
+import com.epam.esm.user.entity.User;
+import com.epam.esm.user.repository.UserRepository;
+import com.github.javafaker.Faker;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.time.LocalDateTime;
+import java.util.*;
+
+@SpringBootApplication(scanBasePackages = "com.epam.esm")
+@EnableJpaRepositories({
+        "com.epam.esm.certificate.repo",
+        "com.epam.esm.tag.repo",
+        "com.epam.esm.user.repository",
+        "com.epam.esm.order.repository"
```

Comment:
This naming looks inconsistent to me.
Also try to use wildcard for all repos scan

Reply:
Because I moved this class outside base folder, it could not see the repositories and entity classes without annotations. Fixed.